steamworks_unix.go

```diff
@@ -151,6 +156,8 @@ func (l *lib) call(ftype funcType, name string, args ...uintptr) (C.uint64_t, er
 		return C.uint64_t(C.callFunc_Bool_Ptr_Ptr_Ptr(f, C.uintptr_t(args[0]), C.uintptr_t(args[1]), C.uintptr_t(args[2]))), nil
 	case funcType_Bool_Ptr_Ptr_Ptr_Int32:
 		return C.uint64_t(C.callFunc_Bool_Ptr_Ptr_Ptr_Int32(f, C.uintptr_t(args[0]), C.uintptr_t(args[1]), C.uintptr_t(args[2]), C.int32_t(args[3]))), nil
+	case funcType_Bool_Ptr_Int32_Int32_Int32_Int32:
+		return C.uint64_t(C.callFunc_Bool_Ptr_Int32_Int32_Int32_Int32(f, C.uintptr_t(args[0]), C.int32_t(args[1]), C.int32_t(args[2]), C.int32_t(args[3]), C.int32_t(args[4]))), nil
```

Comment:
Isn't `args[5]` ignored?

Reply:
Nice catch. I got lost in the arguments.